client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -63,7 +64,7 @@ const NotesComponent = (props: ArrayControlCustomProps) => {
 
   const options = props.uischema.options;
 
-  const inputData = data ?? [];
+  const inputData = (data as unknown as NoteSchema[]) ?? [];
```

Comment:
Why the double `as`? It compiles fine with just `as NoteSchema[]`.

Reply:
thanks! I removed it, I think I made a different change to make this unnecessary...